tools/rdm/TestMixins.py

```diff
@@ -317,6 +317,9 @@ class SetWithDataMixin(ResponderTestFixture):
   def Test(self):
     results = [
       self.NackSetResult(RDMNack.NR_FORMAT_ERROR),
+      # TODO(Peter): Fix this, ideally we change behaviour based on past
+      # support of the PID
+      self.NackSetResult(RDMNack.NR_UNSUPPORTED_COMMAND_CLASS),
```

Comment:
Are you happy with that as an explanation @nomis52, or do we need a different plan going forward?